src/main/java/org/opensearch/ad/util/Bwc.java

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
just wondering, do you think it's worth keeping this file around just as skeleton code? I imagine we'll need to re-create something like this in the future for other breaking changes that require conditional testing based on the version

Reply:
We can refer to older versions or this PR to bring it back. Having a skeleton code there will make the coverage 0 and will fail our coverage check tool.

Reply:
Fair enough, I'm ok to remove